app/src/main/res/layout/revision_card_fragment.xml

```diff
@@ -76,6 +76,21 @@
             android:text="@{viewModel.explanationLiveData}"
             android:textColor="@color/oppiaPrimaryText"
             android:textSize="16sp" />
+
+          <Button
+            android:id="@+id/revision__card_return_button"
+            style="@style/StateButtonActive"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="20dp"
+            android:layout_marginEnd="28dp"
+            android:background="@drawable/state_button_primary_background"
+            android:text="Return to topic"
```

Comment:
This should be mentioned in `strings.xml`

Reply:
Done